gui-libs/libadwaita/libadwaita-1.7.0.ebuild

```
@@ -0,0 +1,67 @@
+# Copyright 2022-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+inherit gnome.org meson python-any-r1 vala virtualx
+
+DESCRIPTION="Building blocks for modern GNOME applications"
+HOMEPAGE="https://gnome.pages.gitlab.gnome.org/libadwaita/ https://gitlab.gnome.org/GNOME/libadwaita"
+
+LICENSE="LGPL-2.1+"
+SLOT="1"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+
+IUSE="+introspection test +vala"
+REQUIRED_USE="vala? ( introspection )"
+
+RDEPEND="
+	>=dev-libs/glib-2.80:2
+	>=gui-libs/gtk-4.17.5:4[introspection?]
+	dev-libs/appstream:=
+	dev-libs/fribidi
+	introspection? ( >=dev-libs/gobject-introspection-1.54:= )
```

```suggestion
	introspection? ( >=dev-libs/gobject-introspection-1.84:= )
```

Based on my crash report, does this require a bump?

---

OK for clarity, you're saying it was all about arch `dev-lang/vala`?

---

Yes. It's not right for us to workaround that in every consumer of gobject-introspection.